homeworks/artem.sokur_Grimdvl/0-intro/README.md

```diff
@@ -1,3 +0,0 @@
-node version - v18.18.1
```

Comment:
Hey, @SerhiiNSN! Auch! Why do you want to delete the work of your teammate? In this pr should be only your changes.